submissions/maksimsergeev97/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
```

Comment:
This file is redundant, also, all files must be in the task folder. Remove this file.